cumulusci.yml

```diff
@@ -499,6 +503,17 @@ flows:
                 options:
                     namespaced: False
 
+    qa_org_2gp:
+        steps:
+            2:
+                flow: config_2gp
+            3:
+                task: None
```

Comment:
@MatthewBlanski since we are overwriting step 2 with a new task, can we also overwrite step 3 with config_qa instead of None?